app/src/components/dashboard/services/ModalSearch.js

```diff
@@ -0,0 +1,41 @@
+"use client";
+import Button from "@/components/common/buttons/Button";
+import SERVICES from "../../lists/services";
```

Comment:
is there a reason not to use the '@' import style?

Reply:
well noted, thanks!! its due to automated imports from vscode

All of them were corrected.